doc/users/getting_started/index.rst

```diff
@@ -41,15 +47,19 @@ Here is a minimal example plot:
    y = np.sin(x)
 
    fig, ax = plt.subplots()
-   ax.plot(x, y)
+   ax.plot(x, y, label="sin(x)")
+   ax.set_title("A Simple Sine Curve")
```

Comment:
We want to keep the initial plot as minimal as possible. Title and legend are definitely out of scope. One could argue about label/ylabel, because basically no real plot can do without. Though I'm still still slightly inclined to also omit it to make the 3 essential core operation stand out #  (i) create figure and Axes (ii) plot data (iii) show the figure.